atom/browser/api/atom_api_screen.cc

```diff
@@ -120,6 +120,9 @@ void Screen::BuildPrototype(
       .SetMethod("getPrimaryDisplay", &Screen::GetPrimaryDisplay)
       .SetMethod("getAllDisplays", &Screen::GetAllDisplays)
       .SetMethod("getDisplayNearestPoint", &Screen::GetDisplayNearestPoint)
+      #if defined(OS_MACOSX)
+      .SetMethod("getMenuBarHeight", &Screen::getMenuBarHeight)
```

Comment:
Please add a simple test for this method that verifies it returns a number to `spec/api-screen-spec.js`